conf/machine/imx8mp-ddr4-evk.conf

```diff
@@ -6,6 +6,10 @@
 
 require include/imx8mp-evk.inc
 
+# FIXME: This machine is not yet supported by u-boot-fslc, so for now
+# use u-boot-imx for mainline.
+IMX_DEFAULT_BOOTLOADER_use-mainline-bsp = "u-boot-imx"
```

Comment:
Please use:

```
IMX_DEFAULT_BOOTLOADER = "u-boot-imx"
```

As it is not conditional.

Reply:
That doesn't work since we are trying to override the default settings in `imx8mp-evk.inc`:

```
IMX_DEFAULT_BOOTLOADER_use-nxp-bsp = "u-boot-imx"
IMX_DEFAULT_BOOTLOADER_use-mainline-bsp = "u-boot-fslc"
```

Reply:
Good point. I'll merge it in this case.